src/components/Coins.jsx

```diff
@@ -511,7 +512,7 @@ function Coins() {
               </div>
             </div>
             <div className="w-1/2 h-auto p-5 ml-7 bg-skin-charts-background-color rounded-md">
-              {!priceVolumeList.length !== 0 &&
+              {priceVolumeList.length !== 0 &&
                 !priceVolumeList.includes(undefined) &&

```

Comment:
Try simplifying this with => !priceVolumeList.some(item => item === undefined || item === null)